src/index.tsx

```diff
@@ -7,7 +7,7 @@ export interface SketchProps {
 }
 
 export interface Sketch {
-  (instance: p5): void;
+  (instance: P5Instance): void;
```

Comment:
This makes sure that when users using typscript, for example, want to implement the `updateWithProps` function then proper typings for that callback will be applied without the need for manual typing. So instead of:

```ts
p5.updateWithProps((props: SketchProps) => {});
```

Someone could just do:

```ts
p5.updateWithProps(props => {});
```

and the types should be correctly inferred.

Reply:
This is a good thing. I guess that was done this way in the previous version. 

👍